### DISCUSSION TITLE
Import folders

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#1538] [Q&A] [answered]
Hi there,

I have a lot of markdown files organized in folders/sub-folders from another markdown note editor.
How can I add these folders to Trillium?

**Comment by zadam** [comment, marked as answer]
ZIP them and import the ZIP file. Trilium will unpack them while keeping the directory structure.

**Reply by hokutonanto** [reply]
Thanks! That worked like a charm.
